src/server/scripts/Commands/cs_reset.cpp

```diff
@@ -54,6 +56,44 @@ class reset_commandscript : public CommandScript
         return commandTable;
     }
 
+    static bool HandleResetArenaSeasonCommand(ChatHandler* /*handler*/, const char* args)
+    {
+        if (!sWorld->getBoolConfig(CONFIG_ARENA_SEASON_IN_PROGRESS))
```

Comment:
some kind of info should be provided when executing this command and arena season in progress